cherrypicks.go

```diff
@@ -340,3 +349,14 @@ func parseCherryTargetBranches(body string) []string {
 	}
 	return matches
 }
+
+func parseCherryTargetBranchesSingleLine(body string) []string {
+	matches := []string{}
+	regex := regexp.MustCompile(`\x60(([[:word:]]+[_\.-]?)+)\x60`)
+	for _, m := range regex.FindAllStringSubmatch(body, -1) {
+		if len(m) > 1 {
+			matches = append(matches, m[1])
```

Comment:
Question:

Can we not simply append all the matches in one go? :smile_cat:

Reply:
Maybe `FindAllString()` will be better here actually? :thinking: 

It is really closer to what we want I think

Reply:
I agree, but we are filtering the branches based on length. If it has length < 2, then we simply drop it, as we do in the original syntax parser function. Don't think golang has any means of filtering arrays like this unless we write our own. Hence, I don't think we are able to append in one go :/

Reply:
Actually, I'm not too sure. `FindAllString` doesn't care about capturing groups (we don't want to include the backticks), and I don't think any functions does apart from those that contains the word `Submatch`. One must loop through the 2d array to retrieve the correct submatch I believe (go also doesnt support lookbehinds and lookaheads for complexity reasons)

Reply:
Probably explains why it was not used  in the first instance :)